Makefile

```diff
@@ -43,6 +43,7 @@ MAKENSIS=makensis
 
 ifeq ($(SUFFIX), .exe)
 all: xca-portable.zip msi-installer-dir.zip
+.PHONY: commithash.h
```

Comment:
Assumed the statement in the commit message was correct.
How does this change cure the issue?

commithash.h is already marked as .PHONY.
Repeating it for windows builds does not change anything.

Reply:
Oops, I forgot a hunk in this commit!

Amended commit with change. New commit is 62a134c036f19cc9490f0f149aac6021169b03e8